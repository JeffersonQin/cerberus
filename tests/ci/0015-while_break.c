int main(void) {
  int n = 0;
  while (1) {
    if (n > 9)
      break;
    n++;
  }
  return n; // 10
}
