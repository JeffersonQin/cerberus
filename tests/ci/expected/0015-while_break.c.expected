Defined {value: "Specified(10)", stdout: "", blocked: "false"}
